README.md

```diff
@@ -17,3 +17,4 @@ exhaustive list, a good place to start assuming your ledger name is `NewLedger`:
 4. Find and replace all `Template Ledger` with `New Ledger`
 5. Find and replace all `template_ledger` directories and files with `new_ledger`
 6. Remove `SampleResource` example files.
+7. Create a `RUBY_GEM_KEY` secret for publishing the GEM.
```

Comment:
This belongs in the repo, right? Can we add more specific instructions for how to put this key in the repo?